src/service/url-replacements-impl.js

```diff
@@ -112,6 +141,11 @@ export class GlobalVariableSource extends VariableSource {
     /** @const {!./viewport/viewport-interface.ViewportInterface} */
     const viewport = Services.viewportForDoc(this.ampdoc);
 
+    // Lazily cache the geo location if available.
```

Comment:
Isn't this greedy instead of lazy? We are caching the service (ideally) well before its use.

```suggestion
    // Greedily cache the geo location if available.
```

Reply:
Yes it is! Good point.